### PR TITLE
Add Stats endpoints, settings and management commands

### PR DESCRIPTION
**Pull request by cduhn17**
#  #

## 🗣 Description ##

Merge the methods to get stats infor for the frontend into the main backend python branch. Also includes includes updates to settings to centralize env variables to be used in API endpoints, tasks that will be run to populate AWS elasticache and supporting management commands that if in the event we need to manually push new data to elasticache

## 💭 Motivation and context ##

This change is required to add functionality to the stats endpoints. 

## ✅ Pre-approval checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] *All* future TODOs are captured in issues, which are referenced
      in code comments.
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

## ✅ Pre-merge checklist ##

- [x] Revert dependencies to default branches.
- [x] Finalize version.

## ✅ Post-merge checklist ##

- [x] Create a release.

### CODE REVIEW

File: backend/src/xfd_django/xfd_django/settings.py
Comment (rapidray12) [inline backend/src/xfd_django/xfd_django/settings.py:1]: Secret key in the source code = No Bueno!!!

Comment (cduhn17) [reply backend/src/xfd_django/xfd_django/settings.py:1]: the key is removed

File: backend/src/tasks/functions.yml [hunk 1]
```diff
@@ -58,3 +58,125 @@ updateScanTaskStatus:
           detail:
             clusterArn:
               - ${file(env.yml):${self:provider.stage}-ecs-cluster, ''}
+
+populateServiceStatsElasticache:
+  handler: src/xfd_django/xfd_api/tasks.populate_ServicesStatscache
```
Comment (aloftus23) [inline backend/src/tasks/functions.yml]: Make sure these paths are going to the correct functions. I don't see a tasks.populate...

File: backend/src/xfd_django/xfd_api/views.py [hunk 2]
```diff
@@ -658,6 +687,510 @@ async def invoke_scheduler(current_user: User = Depends(get_current_active_user)
     return response
 
 
+@api_router.get(
+    "/services/",
+    tags=["Retrieve Stats"],
+)
+async def get_services(
```
Comment (aloftus23) [inline backend/src/xfd_django/xfd_api/views.py:673]: See other endpoint definitions. We want these as minimal as possible so move to a function in api_methods/

File: backend/src/tasks/functions.yml (hunk 1 above)
Comment (cduhn17) [reply backend/src/tasks/functions.yml]: I have changed the location of the tasks per commit [2136652](https://github.com/cisagov/XFD/pull/707/commits/21366526b34d5de1b8e649cc4c98f2b2e19e7c03)

File: backend/src/xfd_django/xfd_api/views.py (hunk 2 above)
Comment (cduhn17) [reply backend/src/xfd_django/xfd_api/views.py:673]: I have altered the directory structure to match all other endpoint and make adjustment to import statements at views.py per commit [2136652](https://github.com/cisagov/XFD/pull/707/commits/21366526b34d5de1b8e649cc4c98f2b2e19e7c03)

**Pull request merged**